meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx_6.6.bb

```diff
@@ -8,20 +8,10 @@ include recipes-kernel/linux/linux-lmp-fslc-imx.inc
 include recipes-kernel/linux/kmeta-linux-lmp-6.6.y.inc
 
 # Use Freescale kernel by default
-LINUX_VERSION ?= "6.6.54"
-KERNEL_BRANCH ?= "6.6-2.1.x-imx"
+LINUX_VERSION ?= "6.6.74"
+KERNEL_BRANCH ?= "6.6-2.2.x-imx"
 
-SRCREV_machine = "6605f549a71ef3af9c65609cba985086ac4edf2b"
-
-SRC_URI += " \
-    file://0004-FIO-toup-hwrng-optee-support-generic-crypto.patch \
-    file://0001-arm64-dts-imx8mq-drop-cpu-idle-states.patch \
-    file://0001-FIO-toimx-of-enable-using-OF_DYNAMIC-without-OF_UNIT.patch \
-    file://0002-FIO-toup-media-Kconfig-fix-double-VIDEO_DEV.patch \
-    file://0003-FIO-toup-gpu-drm-cadence-select-hdmi-helper.patch \
-    file://0004-FIO-toup-media-imx8-select-v4l2_-for-mxc-mipi-csi2_y.patch \
-    file://0001-imx-dts-imx8mm-evkb-fix-the-pmic-name-to-avoid-dupli.patch \
```

Comment:
All the patches can also be removed from the layer

Reply:
Did we upstream all the patches? AFAIK 0004-FIO-toup-hwrng-optee-support-generic-crypto.patch is a downstream one.

Reply:
Oh, great then :-) Just need to remove from the layer then.

Reply:
I just reviewed the list, there is only one patch that I haven't sent to linux-fslc, which is the one removing the double line for depends VIDEO_IMX (at that time, I thought the patch was not needed), but I'm sending it now